services/py-api/tests/integration_tests/test_participant_routes.py

```diff
@@ -0,0 +1,138 @@
+from src.database.repository.participants_repository import ParticipantsRepository
+import pytest
+from httpx import AsyncClient
+from src.database.repository.teams_repository import TeamsRepository
+
+PARTICIPANT_ENDPOINT_URL = "/api/v3/hackathon/participants"
+
+RANDOM_PARTICIPANT_BODY = {"name": "testtest", "email": "testtest@test.com", "is_admin": False}
+IS_ADMIN_TRUE_BODY = {"name": "testtest", "email": "testtest@test.com", "is_admin": True}
+MISSING_REQ_FIELDS_BODY = {"name": "testtest", "is_admin": False}
+
+ADMIN_PARTICIPANT_BODY = {"name": "testtest", "email": "testtest@test.com", "is_admin": True, "team_name": "testteam"}
+EXISTING_TEAM_NAME_BODY = {"name": "testtest", "email": "testtest1@test.com", "is_admin": True, "team_name": "testteam"}
+EXISTING_EMAIL_BODY = {"name": "testtest", "email": "testtest@test.com", "is_admin": True, "team_name": "testteam1"}
+
+
+@pytest.mark.asyncio(loop_scope="session")
+async def test_create_random_participant(
+    async_client: AsyncClient, participant_repository: ParticipantsRepository
+) -> None:
+    resp = await async_client.post(PARTICIPANT_ENDPOINT_URL, json=RANDOM_PARTICIPANT_BODY)
+
+    # Perform cleanup
+    await participant_repository.delete(resp.json()["participant"]["id"])
+
+    assert resp.status_code == 201
+    assert resp.json()["participant"]["name"] == "testtest"
+    assert resp.json()["participant"]["email"] == "testtest@test.com"
+    assert resp.json()["participant"]["is_admin"] == False
+    assert resp.json()["participant"]["email_verified"] == False
+    assert resp.json()["participant"]["team_id"] == "None"
+    assert resp.json()["team"] == None
+
+
+@pytest.mark.asyncio(loop_scope="session")
+async def test_create_random_participant_email_already_exists(
+    async_client: AsyncClient, participant_repository: ParticipantsRepository
+) -> None:
+
+    resp1 = await async_client.post(PARTICIPANT_ENDPOINT_URL, json=RANDOM_PARTICIPANT_BODY)
+    resp2 = await async_client.post(PARTICIPANT_ENDPOINT_URL, json=RANDOM_PARTICIPANT_BODY)
+
+    # Performing cleanup, only the first response is successful
+    await participant_repository.delete(resp1.json()["participant"]["id"])
```

Comment:
I would suggest having the `delete` endpoint; the Auth Middleware is easy to implement (We could even use a hardcoded token auth bearer for now). Currently, we don't have these changes deployed, so it's not a problem. So please change the tests to use the `delete` endpoint, as we would need to change these tests in the future. Why bother? Also, I don't think the integration tests should bother with creating real instances of these repositories. As you said in the PR description, they are already created when making a request via the `AsyncClient`.

Reply:
I mean we dont neccessarily need to rewrite the tests, but I see your point. I though of just having a token and writing a auth middleware but it seemed like too much work ahahhaha, however I trust you on this one.

Reply:
Fixed in a10d828544025ee80ad7d36e5033831436a2679a